Fancy Hats/assets/css/styles.css

```diff
@@ -0,0 +1,136 @@
+/* UNIVERSAL STYLES */
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
```

Comment:
I dont see OpenSan, helvetica fonts imports

Reply:
Now You can see the font family  which i gave. (I added the fonts there in CSS itself and it works without import)